stryker.conf.json

```diff
@@ -16,6 +16,9 @@
   },
   "coverageAnalysis": "all",
   "commandRunner": {
-    "command": "npm run test:unit"
-  }
+    "command": "jest"
+  },
+  "files": [
+    "test/**/*.test.js", "test/**/*.xml", "test/**/*.html", "test/createParser.js", "lib/*.js", "package.json", "jest.config.js"
+  ]
```

Comment:
I think these changes are not needed. We should run Stryker with the entire test suite.

Reply:
This was here for two reasons:
- improve performance of copying into `.stryker-tmp` folders (I don't think it's needed anymore)
- My expectation is that the xmltest suite covers everything (and more) that the vows testsuite does. I'll double check. If the current score of 99.7% is increased by also running the vows tests as part of stryker, I would of course include them. 

That being said, using npm to run jest and vows will be a major performance drain for running stryker.

Reply:
The result is outside of how I understood stryker so far:

stryker command | Mutation score | # Killed | # Survived | # Timeout | # No coverage | # Ignored | # Runtime errors | # Compile errors | Total detected | Total undetected | Total mutants
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
`jest` | 99.70% | 1958 | 6 | 35 (timeoutMS:15s) | 0 | 0 | 0 | 0 | 1993 | 6 | 1999
`npm run test:unit` | 64.73% | 1243 | 705 | 51 (timeoutMS:20s) | 0 | 0 | 0 | 0 | 1294 | 705 | 1999

The higher amounts of timeout is most likely related to tests taking to long (I have see this number go up drastically when the timeout it to low.
But why would more tests result in less coverage?

I will try to see if that is an issue with `npm-run-all` ...